app/src/main/java/com/android/derekbearded/Sunshine/Util.java

```diff
@@ -1,58 +1,50 @@
 package com.android.derekbearded.Sunshine;
 
-import com.squareup.moshi.JsonAdapter;
-import com.squareup.moshi.JsonReader;
-import com.squareup.moshi.Moshi;
-
 import java.io.IOException;
+import java.util.List;
 
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.moshi.MoshiConverterFactory;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
 
 /**
  * Created by Sputnik on 3/9/2018.
  */
 
 public class Util {
```

Comment:
I know this class is called Util, but it's really the API/Service/Repository layer. We would call it an "interactor". A layer that the presenter uses to access the API/Service/Repository, and handles any mapping/data massaging to get the response into a format the presenter requires.